eventhubs/spring-cloud-azure-stream-binder-eventhubs/eventhubs-binder/README.md

```diff
@@ -15,206 +17,257 @@ Running this sample will be charged by Azure. You can check the usage and bill a
     spring:
       cloud:
         azure:
-          msi-enabled: true
-          client-id: [the-id-of-managed-identity]
-          resource-group: [resource-group]
-          # Fill subscription ID copied from portal
-          subscription-id: [subscription-id]
-          eventhub:
-            namespace: [eventhub-namespace]
-            checkpoint-storage-account: [checkpoint-storage-account]
-            checkpoint-container: [checkpoint-container]
+          credential:
+            managed-identity-client-id: ${AZURE_MANAGED_IDENTITY_CLIENT_ID}
+          profile:
+            tenant-id: ${AZURE_TENANT_ID}
+          eventhubs:
+            namespace: ${AZURE_EVENTHUBS_NAMESPACE}
+            processor:
+              checkpoint-store:
+                container-name: ${AZURE_STORAGE_CONTAINER_NAME}
+                account-name: ${AZURE_STORAGE_ACCOUNT_NAME}
         stream:
           function:
             definition: consume;supply
           bindings:
             consume-in-0:
-              destination: [eventhub-name]
-              group: [consumer-group]
+              destination: ${AZURE_EVENTHUB_NAME}
+              group: ${AZURE_EVENTHUB_CONSUMER_GROUP}
             supply-out-0:
-              destination: [the-same-eventhub-name-as-above]
+              destination: ${SAME_AS_ABOVE_DESTINATION}
+          eventhubs:
+            bindings:
+              consume-in-0:
+                consumer:
+                  checkpoint:
+                    mode: MANUAL
+          default:
+            producer:
+              errorChannelEnabled: true
+          poller:
+            initial-delay: 0
+            fixed-delay: 1000
     ```
-    > We should specify `spring.profiles.active=mi` to run the Spring Boot application. 
-      For App Service, please add a configuration entry for this.
+   
+    > We should specify `spring.profiles.active=mi` to run the Spring Boot application.
+    For App Service, please add a configuration entry for this.
 
 ##### Redeploy Application
 
-If you update the `spring.cloud.azure.managed-identity.client-id`
+If you update the `spring.cloud.azure.credential.managed-identity-client-id`
 property after deploying the app, or update the role assignment for
 services, please try to redeploy the app again.
 
-> You can follow 
-> [Deploy a Spring Boot JAR file to Azure App Service][deploy-spring-boot-application-to-app-service] 
+> You can follow
+> [Deploy a Spring Boot JAR file to Azure App Service][deploy-spring-boot-application-to-app-service]
 > to deploy this application to App Service
 
-#### Enable auto create
-
-If you want to auto create the Azure Event Hub and Azure Storage account instances,
-make sure you add such properties 
-(only support the service principal and managed identity cases):
-
-```yaml
-spring:
-  cloud:
-    azure:
-      subscription-id: [subscription-id]
-      auto-create-resources: true
-      environment: Azure
-      region: [region]
-```
-
 #### Enable sync message
```

Comment:
This section should be placed in the section of "Enhancement"

Reply:
updated